### DISCUSSION TITLE
Suggestion about adding custom shortcut in VSCode

### DISCUSSION THREAD

**Discussion by marcuswang6** [quarto-dev/quarto-cli#3399] [Q&A]
There are many regular commands to use, but no shortcuts like below in VSCode:

`Bold` = `Cmd+B`
`Italic` = `Cmd+I`
`Create Python Code Snippets` = `Cmd+N`
`Heading 1` = `Cmd+1`

Can we just customize these functions by ourselves?

**Comment by mcanouil** [comment]
Yes, create your own snippet and associate it with any keyboard shortcuts. For more, please take a look at VSCode documentation.
To note, by default, if you select text and press `*`, then the text is surrounded by `*`.
A lot of formatting functions/snippets are accessible via the command palette, so check it out first before reinventing the wheel.
